pkg/ottl/ottlfuncs/README.md

```diff
@@ -316,6 +319,28 @@ Examples:
 
 - `ConvertCase(metric.name, "snake")`
 
+### FNV
+
+`FNV(value)`
+
+The `FNV` Converter converts the `value` to an FNV hash/digest.
+
+The returned type is int64.
+
+The input `value` types:
+* string. If it fails then nil will be returned.
+
+If `value` is another type or parsing failed nil is always returned.
+
+The `value` is either a path expression to a telemetry field to retrieve or a literal.
```

Comment:
```suggestion
`value` is either a path expression to a string telemetry field or a literal string. If `value` is another type an error is returned.

If an error occurs during hashing it will be returned.
```